src/shared-components/Button/Button.module.css

```
@@ -103,9 +103,33 @@
   cursor: not-allowed;
 }
 
+/* ── Toolbar action button variant ───────────────────────────────────────────── */
+:global(.btn-toolbar-action) {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)